packages/fxa-auth-server/lib/geodb.js

```diff
@@ -42,10 +42,12 @@ module.exports = log => {
         confidence += 'no_accuracy_data';
       }
 
-      log.info('geodb.accuracy', { accuracy });
-      log.info('geodb.accuracy_confidence', {
-        accuracy_confidence: confidence,
-      });
+      if (config.logAccuracy) {
+        log.info('geodb.accuracy', { accuracy });
```

Comment:
This was used to create a kibana dashboard which we don't have access anymore. Welcome to rip out the log statements too.

Reply:
Okay. I'll leave the lines as an option, but defaulted to off.